<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addClients</class>
 <widget class="QDialog" name="addClients">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dodawanie nowego kontrahenta</string>
  </property>
  <widget class="QLineEdit" name="output_city">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>74</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="output_KRS">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>164</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="output_phone">
   <property name="geometry">
    <rect>
     <x>159</x>
     <y>253</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="output_email">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>253</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>PESEL:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>145</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>KRS:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="output_name">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>28</y>
     <width>284</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="output_NIP">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>74</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="output_REGON">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>119</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>145</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>numer:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ulica:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>159</x>
     <y>234</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>telefon:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_return">
   <property name="geometry">
    <rect>
     <x>216</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Wróć</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>234</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>email:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>REGON:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="output_postalCode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>209</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="output_PESEL">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>209</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="output_number">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>164</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>55</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>miasto:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>nazwa firmy (klienta):</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_add">
   <property name="geometry">
    <rect>
     <x>133</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Dodaj</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>kod pocztowy:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="output_street">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>119</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>55</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>NIP:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
